R/format_ae_specific.R

```diff
@@ -68,30 +68,25 @@ format_ae_specific <- function(outdata,
 
   # Define total column
   display_total <- "total" %in% display
-  index_total <- ncol(outdata$n)
+  index_total <- seq(ncol(outdata$n) - (! display_total) )
+
+  # Drop total column from outdata if it's not requested.
+  outdata$group <- outdata$group[index_total]
```

Comment:
Need to tell `format_ae_specific()` to drop the `total` from `outdata$group` if not needed.